From 8d2f8247f9cb800db708bb26fccbc7b8b0249fcc Mon Sep 17 00:00:00 2001
From: Andrei Stefanescu <andrei.stefanescu@nxp.com>
Date: Tue, 6 Sep 2022 15:54:44 +0300
Subject: [PATCH 01/49] plat: s32g274aemu: disable ERRATA_050543

Since the errata is for an issue with the DDR we don't necessarily need
to apply it to the s32g274aemu target. This commit removes it because
the support for the errata is only implemented in the other driver.

Issue: ALB-9146

Upstream-Status: Pending 

Signed-off-by: Andrei Stefanescu <andrei.stefanescu@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 plat/nxp/s32/s32g/s32g2/s32g274aemu/platform.mk | 2 ++
 plat/nxp/s32/s32g/s32g3/s32g399aemu/platform.mk | 2 ++
 2 files changed, 4 insertions(+)

diff --git a/plat/nxp/s32/s32g/s32g2/s32g274aemu/platform.mk b/plat/nxp/s32/s32g/s32g2/s32g274aemu/platform.mk
index 29331ba1b..790d2ee8b 100644
--- a/plat/nxp/s32/s32g/s32g2/s32g274aemu/platform.mk
+++ b/plat/nxp/s32/s32g/s32g2/s32g274aemu/platform.mk
@@ -13,3 +13,5 @@ S32_BOARD_PATH		:= ${PLAT_SOC_PATH}/s32g274aemu
 PLAT_INCLUDES		+= -I${S32_BOARD_PATH}/include \
 
 DTB_FILE_NAME		?= s32g274a-emu.dtb
+
+ERRATA_S32_050543	:= 0
diff --git a/plat/nxp/s32/s32g/s32g3/s32g399aemu/platform.mk b/plat/nxp/s32/s32g/s32g3/s32g399aemu/platform.mk
index 75368cda7..9401874b3 100644
--- a/plat/nxp/s32/s32g/s32g3/s32g399aemu/platform.mk
+++ b/plat/nxp/s32/s32g/s32g3/s32g399aemu/platform.mk
@@ -13,3 +13,5 @@ S32_BOARD_PATH		:= ${PLAT_SOC_PATH}/s32g399aemu
 PLAT_INCLUDES		+= -I${S32_BOARD_PATH}/include \
 
 DTB_FILE_NAME		?= s32g399a-emu.dtb
+
+ERRATA_S32_050543	:= 0
-- 
2.25.1

